docs/dev/testing.md

```diff
@@ -0,0 +1,34 @@
+# Running the tests
+
+## Running all tests
+
+The tests for all languages my be run from the command line (on Unix platforms):
```

Comment:
```suggestion
The tests for all languages may be run from the command line (on Unix platforms):
```

Reply:
Also, this works on Windows as well.

Reply:
Oh, interesting.  I vaguely recall @Dexterp37 reporting that it didn't...

Reply:
Works in my mozillabuild shell (just set it up yesterday). Maybe it depends what env you're running it in...

I've been keeping notes on what it's taken to get up and running. Maybe Alessio and I need to compare Windows notes.

Reply:
Yup. I'm not considering `mozillabuild` shell a requirement for this project to be honest. My *personal* definition of "working on Windows" is you can open a command prompt, call commands in it, and it will just work.

By default, as far as I know, Windows doesn't know what a `Makefile` is, so can't run that out of the box. However, the commands executed by `make test` work just fine  when executing in the usual Windows command prompt.

An alternative solution is to use the Linux subsystem for Windows (aka bash for Windows). `make test` will work there, without requiring us an implicit dependency on `mozillabuild`